aura/chain.json

```diff
@@ -37,6 +40,13 @@
     "compatible_versions": [
       "v0.7.1"
     ],
+    "cosmos_sdk_version": "0.47.4",
```

Comment:
can we please add these details to the version obejct?
[Here's ](https://github.com/aura-nw/chain-registry/pull/2)a PR that does that if it helps

Reply:
thanks, merged it.